packages/manager/apps/ips/src/pages/order/sections/OrganisationSection.component.tsx

```diff
@@ -25,37 +31,58 @@ export const OrganisationSection: React.FC = () => {
   const { t } = useTranslation('order');
   const { organisations, isLoading } = useGetOrganisationsList();
 
+  const noOrganisationInfoLabel = useMemo(() => {
+    if (selectedOrganisation) {
+      return t('organisation_selection_info');
+    }
+    if (!selectedOrganisation && isRegionInEu(selectedRegion)) {
+      return t('organisation_selection_description_no_organisation_RIPE');
+    }
+    if (!selectedOrganisation && !isRegionInEu(selectedRegion)) {
+      return t('organisation_selection_description_no_organisation_ARIN');
+    }
+    return undefined;
+  }, [selectedOrganisation, selectedRegion]);
+
+  const handleOrganisationChange = (
+    event: OdsSelectCustomEvent<OdsSelectChangeEventDetail>,
+  ) => {
+    const updatedOrganisation =
+      event.detail.value !== NO_ORGANISATION ? event.detail.value : undefined;
+    setSelectedOrganisation(updatedOrganisation);
+  };
+
   return (
     <OrderSection
       title={t('organisation_selection_title')}
-      description={
-        t('organisation_selection_description') +
-        (isRegionInEu(selectedRegion)
-          ? t('organisation_selection_description_no_organisation_RIPE')
-          : t('organisation_selection_description_no_organisation_ARIN'))
-      }
+      description={t('organisation_selection_description')}
     >
-      <OdsMessage
-        isDismissible={false}
-        className="block mb-3"
-        color={ODS_MESSAGE_COLOR.information}
-      >
-        {t('organisation_selection_info')}
-      </OdsMessage>
+      {noOrganisationInfoLabel ? (
+        <OdsMessage
+          isDismissible={false}
+          className="block mb-3"
+          color={ODS_MESSAGE_COLOR.information}
+        >
+          {noOrganisationInfoLabel}
+        </OdsMessage>
+      ) : (
+        undefined
+      )}
```

Comment:
```suggestion
        <OdsMessage
          isDismissible={false}
          className="block mb-3"
          color={ODS_MESSAGE_COLOR.information}
        >
          {noOrganisationInfoLabel}
        </OdsMessage>
```

If it's not possible to not have a label